test/utils.test.js

```diff
@@ -36,3 +36,40 @@ describe("Find Highest of Numbers findHighest(...numbs)", () => {
     expect(findHighest(3,6,2,4,1)).to.equal(6);
   })
 })
+
+describe("Find Average of Numbers findAverage(...numbers)", () => {
```

Comment:
Let's add tests for when we expect errors/exceptions to be thrown.

Reply:
noted